applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -133,6 +146,8 @@
               ,prev = <<"4">> :: kz_term:ne_binary()
               ,next = <<"6">> :: kz_term:ne_binary()
               ,delete = <<"7">> :: kz_term:ne_binary()
+              ,rw = <<"5">> :: kz_term:ne_binary()
```

Comment:
we try not to abbreviate names - please use `rewind` and `fast_forward` (esp since `rw` makes me think file I/O `read/write`).